Gemfile.lock

```diff
@@ -86,6 +86,8 @@ GEM
       dry-configurable (~> 0.1, >= 0.1.3)
     dry-core (0.5.0)
       concurrent-ruby (~> 1.0)
+    email_validator (2.2.2)
```

Comment:
@josephelias-rubyians  How is being used for validating email? When you add gems, you need to have code to show how its being applied.

Reply:
In the gem doc they are given like this " validates :my_email_attribute, email: true "
I have used the same in our code too.
https://github.com/K-and-R/email_validator